pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -821,6 +822,12 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 				networkNameExistsOrNotNeeded = network.Genie.NetworkName != ""
 			}
 
+			if network.NetworkSource.Kuryr != nil {
+				cniTypesCount++
+				kuryrExists = true
+				networkNameExistsOrNotNeeded = network.Kuryr.NetworkName != ""
```

Comment:
not a part of this PR, but this variable does not make sense with the error message "CNI delegating plugin must have a networkName"

Reply:
why? this variable is "true" in one of two cases: network name exists, or not needed.
if the network name is needed, but not set, we gave the above error